harvester/core/models/datatypes/arrangement.py

```diff
@@ -3,13 +3,11 @@
 from django.db import models
 from django.contrib.postgres import fields as postgres_fields
 
-from datagrowth.datatypes import CollectionBase, DocumentCollectionMixin
-
 
 logger = logging.getLogger("harvester")
 
 
-class Arrangement(DocumentCollectionMixin, CollectionBase):
+class Arrangement(object):
```

Comment:
is deze file nu nog wel nodig?

Reply:
Er wordt nog gerefereerd naar Arrangement vanuit de migrations. Als het straks overal weg is, kunnen we het uit de migrations halen. Wat ook kan is dat ik het weghaal en even kijken of de migrations gewoon goed runnen met een schone installatie. Ik heb er weleens problemen mee gehad, maar er wordt alleen naar Arrangement gerefereerd in een string literal. Geen imports ofzo, dus dan zou je hopen dat Arrangement gewoon weg kan.

Reply:
Ja, of een definitie van die class in de migratie. Ik weet niet voor hoeveel plekken het geldt.

Reply:
Lijkt gewoon te werken. Ik denk dat de vorige develop partij wat vreemde dingen in de migrations heeft gedaan. Dat is een tijdje terug allemaal weg gesquashed :partying_face:

Reply:
Fijn!